component-info.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: "kafka-connect-plugins"
+  description: "A set of generic plugins for Kafka Connect that complement the built-in transformations, config providers, and connectors."
+  links: # add any links that would be useful here. Dashboards, site urls etc
+    - title: Source
+      url: https://github.com/cultureamp/kafka-connect-plugins
+  tags:
+    - tier-1
```

Comment:
Is tier-1 the right tag?
Any other tags that should be here?

Reply:
So I'm basing this off the catalog file in vault-ops, and there was this `users-internal` tag listed, but because this repo is public I've taken that out. Let me know if there's another tag that should go on this file.